recovery/recovery.go

```diff
@@ -226,6 +259,25 @@ func mountFilesystem(label string, mountpoint string) error {
 	return nil
 }
 
+func storeKeyfile(dir string, buffer []byte) error {
+	// TODO: seal keyfile
+	if err := ioutil.WriteFile(path.Join(dir, "keyfile"), buffer, 0400); err != nil {
+		return err
+	}
+
+	// Don't remove this sync, it prevents file corruption on vfat
+	if err := exec.Command("sync").Run(); err != nil {
+		return fmt.Errorf("cannot sync: %s", err)
```

Comment:
Please just use `syscall.Sync` instead. No need to fork for this.

Reply:
Ah, excellent, that's much better, thank you.